labels.md

```diff
@@ -0,0 +1,220 @@
+component/metrics               #e26f88     component/metrics                   #d1fad7
+component/mysql-protocol        #66dde8     component/MySQL protocol            #d1fad7
+component/parser                #fbca04     component/parser                    #d1fad7
+component/planner               #1d76db     component/planner                   #d1fad7
+component/plugin                #c98fe8     component/plugin                    #d1fad7
+component/privilege             #81d5e2     component/privilege                 #d1fad7
+component/server                #e0854c     component/server                    #d1fad7
+component/session               #49309b     component/session                   #d1fad7
+component/statistics            #80e58d     component/statistics                #d1fad7
+component/syncer                #ccbf0e     component/syncer                    #d1fad7
+component/test                  #6494f4     component/test                      #d1fad7
+component/tikv                  #8bef73     component/tikv                      #d1fad7
+component/tools                 #cbf232     component/tools                     #d1fad7
+component/transaction-need-LGT  #83e2d1     component/transaction (3lgtm)       #d1fad7
+component/usability             #0ad8bd     component/usability                 #d1fad7
+component/util                  #47e0b0     component/util                      #d1fad7
+contribution                    #b0ff5b     type/contributor ⭐️                 #1d76db
+errmsg                          #a7c938     type/error message                  #1d76db
+for new contributors            #c2e0c6     difficulty/Mentor                   #0e8a16
+help wanted                     #159818     status/help wanted                  #e6e6e6
+needs-cherry-pick-2.1           #000000     type/cherry pick 2.1                #1d76db
+needs-cherry-pick-3.0           #000000     type/cherry pick 3.0                #1d76db
+priority/P1                     #b60205     priority/low                        #eb6420
+priority/P2                     #FF7F50     priority/medium                     #0e8a16
+priority/non-release-blocker    #0052cc     priority/high                       #eb6420
+priority/release-blocker        #b60205     priority/blocker                    #eb6420
+proposal                        #eb6420     type/proposal                       #1d76db
+question                        #fbca04     type/question                       #1d76db
+release-note                    #1d76db     * <type/CHANGELOG>                    
+require-LGT3                    #d93f0b     type/require 3 reviews              #1d76db
+status/DNM                      #b60205     status/DNM                          #e6e6e6
+status/LGT1                     #d4c5f9     * <status/Waiting on review>            
+status/LGT2                     #5319e7     * <status/Waiting on review>            
```

Comment:
I generally like the whole idea of having a standard label system for all repos and the way to categorize labels. But I do have a question about letter cases, looks like most of them are using all lower cases except some. Are we going to make them consistent?

Reply:
They should all be consistent. The `* < ... >` mean that this label should be deleted and to use the label in the `< ... >` instead, so some of those I might have missed the capitalisation, but it doesn't matter so much. Acronyms are all caps, names are capitalised, everything else should be lower case (i.e., everything is sentence case).

Reply:
Cool. It all looks good to me.